examples/decoding/plot_receptive_field.py

```diff
@@ -65,34 +64,41 @@
 ###############################################################################
 # Create and fit a receptive field model
 
-# Define the delays that we'll use in the receptive field
+# Define the delays that we will use in the receptive field
 tmin, tmax = -.4, .2
 
 # Initialize the model
 spec_names = ['freq_%s' % ii for ii in range(speech.shape[0])]
-mod = Ridge()
-rf = ReceptiveField(tmin, tmax, sfreq, spec_names)
-delays_time = rf.delays / float(sfreq)
+rf = ReceptiveField(tmin, tmax, sfreq,
+                    feature_names=spec_names, model=Ridge(alpha=1.))
+n_delays = int((tmax - tmin) * sfreq) + 2  # +2 to account for 0 + end
+
+n_splits = 3
+cv = KFold(n_splits)
 
-n_cv = 3
-kf = KFold(n_cv)
+Y, _ = raw[:]  # Outputs for the model
 
 # Iterate through folds, fit the model, and predict/test on held-out data
-coefs = np.zeros([n_cv, len(raw.ch_names), len(delays_time)])
-scores = np.zeros([n_cv, len(raw.ch_names)])
-for ii, (tr, tt) in enumerate(kf.split(speech.T)):
+coefs = np.zeros([n_splits, n_channels, n_delays])
+scores = np.zeros([n_splits, n_channels])
+for ii, (train, test) in enumerate(cv.split(speech.T)):
     print('CV iteration %s' % ii)
-    for jj, i_ch in enumerate(raw[:][0]):
-        rf.fit(speech[..., tr], i_ch[tr])
-        preds = rf.predict(speech[..., tt]).squeeze()
-        scores[ii, jj] = np.corrcoef(preds, i_ch[tt][rf.mask_pred])[1, 0]
-        coefs[ii, jj] = rf.coef_
+    rf.fit(speech[..., train], Y[..., train])
```

Comment:
sorry to insist but coming from sklearn it's weird that samples are selected using second dimension. Would be ok to refactor to just write:

rf.fit(speech[train], Y[train])

?